docs/usage/ruby.md

```diff
@@ -25,3 +25,23 @@ If you always want to have the latest available version, consider specifying `ge
 
 You can install the [Renovate App](https://github.com/apps/renovate) on GitHub.
 Or you can check out [Renovate OSS](https://github.com/renovatebot/renovate) to self-host Renovate.
+
+## Ruby on Rails
+
+Renovate doesn't support [Rails](https://rubyonrails.org/) gems correctly.
+This is because Rails's gems must be updated in a batch.
+
+For example, say you have these Rails gems in your Gemfile:
+
+```
+gem 'rails'
+gem `railties`
```

Comment:
This one should have single quotes instead of backticks.

Reply:
```suggestion
gem 'railties'
```

I use backticks all the time, but here I should use single quotes! 🙈